MageOS/AsyncEventsGCP/README.md

```diff
@@ -1 +1,40 @@
-# MageOS Events integration for GCP
+# MageOS Async Events GCP
+
+GCP event sinks for [mageos-async-events](https://github.com/mage-os/mageos-async-events)
+
+## Installation
+
+```sh
+composer require mage-os/mageos-async-events-aws
+```
+
+## GCP event sinks
+
+### Google Pub/Sub
+
+**Setup Application Default Credentials (ADC)**
+
+A Service Account with the `Pub/Sub Publisher` role is required so that the notifier can relay events into Google
+Pub/Sub.
+
+Under `Stores -> Services -> Async Events GCP` set the `ADC Path`.
```

Comment:
Can you explain this a little more? What is this file for? Where do you get the file from? Where do you typically put it?

I guess it's more or less obvious if you are a seasoned GCP user, but if you are just starting off with it, I'd say a little more guidance would be nice.

Reply:
Sure thing. I searched through the internet for some examples but the only module I could find was GCP bucket storage which unfortunately used a different with mechanism. Re ADC I'll be able to provide some basic documentation and redirect the user to google's documentation.

Regarding the location, I don't have any idea, but I assume any absolute path on the server should be good? I'm new to using Google Cloud as well